Samples/GitHubLabeler/GitHubLabeler/App.config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+  <appSettings>
+    <!--TODO: Please enter your own credentials here. -->
```

Comment:
We should add a warning to this such that people know that they should not commit this file to a public repo, once changed.

Reply:
How far would adding this to the .gitignore go for helping folks to not commit this back to a repo? I've done regex checking in githooks before (eg: `/(password|access_token) ?=/i`), but I don't think githooks can be added to a repo & auto-run by other users.

Reply:
I don't think there is a great solution for this. In general, the recommendation is to avoid using credentials from code and instead use certificates, user-based authentication, or a service like Azure KeyVault. Unfortunately this doesn't work for samples very well...